### DISCUSSION TITLE
category/java%E6%A0%B8%E5%BF%83/

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#3] [Announcements]
# category/java%E6%A0%B8%E5%BF%83/

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

http://localhost:8080/category/java%E6%A0%B8%E5%BF%83/?page=4

**Comment by itwanger** [comment]
只恨自己没文化，一句“二哥牛逼走天下”。
